shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.js

```diff
@@ -225,17 +325,16 @@ const setNoticeForCase = async ({
 
   caseEntity.addDocketEntry(standingPretrialDocketEntry);
 
-  const servedParties = aggregatePartiesForService(caseEntity);
-
+  // TODO: consider moving these two (four) calls to below `serveNoticesForCase`
```

Comment:
Nit: can this be removed?

Reply:
Sorry y'all i didn't see @rachelschneiderman's comments